tests/test_autonomy/test_cli/test_analyse/test_abci/test_check_handlers.py

```diff
@@ -34,9 +34,6 @@
 from tests.test_autonomy.test_cli.base import BaseCliTest
 
 
-@pytest.mark.skipif(
-    platform.system() == "Windows", reason="Fix path resolutions on windows."
-)
 class TestCheckHandlers(BaseCliTest):
```

Comment:
What about `**/skill.yaml`?

Reply:
Isn't it safe since it is `pathlib`'s `Path.glob`?

Reply:
Dunno - can you confirm?

Reply:
Just confirmed by using:
```python
import tempfile
from pathlib import Path


p = Path(tempfile.gettempdir())
print(list(p.glob("**/an_existing_folder_in_the_temp_dir")))
```

and it returned non-empty `list`.